### PR TITLE
fix legal html cruft

### LINKED ISSUE #710: Privacy Notice legal page has unprocessed "<a>Learn More.</a>" blocks
Found in http://testpilot.dev.mozaws.net/__version__ (ca8af43986ea2e0f8f8ecb08625b26f1aae7a6e5):

``` json
{ "commit": "ca8af43986ea2e0f8f8ecb08625b26f1aae7a6e5",
  "version": "",
  "source": "https://github.com/mozilla/testpilot",
  "build": "https://circleci.com/gh/mozilla/testpilot/324" }
```

---

http://testpilot.dev.mozaws.net/privacy

test_pilot

We either need to add some jQuery or other sauce to get toggling working, or take the cheap way out, and just do a simple search/replace of "<a>Learn More.</a>" in the legal templates before/after converting to HTML.

### PR DESCRIPTION
**Pull request by johngruen**
Fixes #710

### CODE REVIEW

**Comment by pdehaan** [comment]
:+1:

**Comment by johngruen** [comment]
@ckprice?

**Comment by ckprice** [comment]
@johngruen do you want to address these here?
- [x] Add in links from [the Terms of Use document](https://docs.google.com/document/d/1e4UiUiqnj2FwbPqB7FN6zbTpsWsasUuaBb9037vuE-I/edit) in sections 2 and 4.
- [x] Update text links to `Terms of Use` instead of `Terms of Service` under the green button.

**Comment by meandavejustice** [comment]
:+1: from me once Cory's comment is addressed.

**Comment by pdehaan** [comment]
https://testpilot.firefox.com/privacy and https://testpilot.firefox.com/terms have footer links of "Privacy" and "Terms of Use"
https://testpilot.firefox.com/ has links of "Privacy" and "Terms"

Related:
firefox_test_pilot_and_include_origin_as_referer_header_in_api_requests_from_addon_by_lmorchard_ _pull_request__661_ _mozilla_testpilot_and_firefox_test_pilot

The homepage says "Terms of Service" and "Privacy Notice", which now seem inconsistent as well (not to mention the order is different in the "By proceeding..." and the footer links -- but that may well be the stupidest comment and observation I've made in the past 2.5 years here).
Not sure if we should just fix the footer links to say "Privacy Notice" and "Terms of Use" globally to match the page headers, or ...

File: legal-copy/terms-of-use.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ May 3, 2016
     “Test Pilot” is Mozilla’s platform to share the next generation of Firefox features (“Test Pilot Experiments”) with volunteers interested in testing, providing feedback, and possibly contributing to code.  Please read this entire document (“Terms”) carefully because it explains your rights and responsibilities when you use Test Pilot or any individual Test Pilot Experiment. By accessing the Test Pilot website, the Test Pilot Add-On, or any Test Pilot Experiment, you agree to be bound by these Terms.
 
 0. ## Account
-    You'll need to have a Firefox Account to participate in Test Pilot. If you sign up for a Firefox Account, you agree to its Terms of Service and Privacy Notice.
+    You'll need to have a Firefox Account to participate in Test Pilot. If you sign up for a Firefox Account, you agree to its [Terms of Service](https://www.mozilla.org/en-US/about/legal/terms/services) and [Privacy Notice](https://www.mozilla.org/en-US/privacy/firefox-cloud).
```
Comment (pdehaan) [inline legal-copy/terms-of-use.md]: We may want to remove the /en-US/ from the URLs.

File: testpilot/frontend/static-src/styles/modules/_legal-information.scss [hunk 2]
```diff
@@ -9,3 +9,9 @@
     text-decoration: underline;
   }
 }
+
+.legal-container h1 {
+  font-size: $font-unit * 3.5;
+  font-weight: 300;
+  margin-top: $grid-unit *2;
```
Comment (pdehaan) [inline testpilot/frontend/static-src/styles/modules/_legal-information.scss]: Not sure if we need a space after the `*` (for sass-lint, maybe).

**Comment by pdehaan** [comment]
@johngruen This needs a rebasin'.
Plus, I think we deleted the *.html files since they were build artifacts.

**Pull request merged**